packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -106,6 +106,11 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   setInitialFocus?: boolean;
 
   /**
+   * Set the max height of Callout component.
+   */
+  maxHeight?: number;
```

Comment:
good catch, chrome is fine, but IE/FF offset are not
![image](https://cloud.githubusercontent.com/assets/25381018/22896794/4476720e-f1d6-11e6-9fe4-4d2c31b7b2c4.png)
Chrome